agent/mibgroup/if-mib/data_access/interface.c

```diff
@@ -32,6 +32,7 @@ netsnmp_feature_require(interface_arch_set_admin_status);
 static netsnmp_conf_if_list *conf_list = NULL;
 static int need_wrap_check = -1;
 static int _access_interface_init = 0;
+static netsnmp_include_if_list *include_list = NULL;
```

Comment:
Please leave out the " = NULL" initializer - it is superfluous. See also https://en.wikipedia.org/wiki/.bss.

Reply:
Will do. Thanks.